common/common-ui/src/main/java/com/duckduckgo/mobile/android/ui/notifyme/NotifyMeView.kt

```diff
@@ -281,6 +281,7 @@ class NotifyMeView @JvmOverloads constructor(
             if (newVisibility != previousVisibility) {
                 this.visibilityChangedListener?.onVisibilityChange(this, newVisibility == VISIBLE)
             }
+            setTag(tag, newVisibility)
```

Comment:
Is this necessary?

Reply:
`setTag()` is done in line 276 with the same parameters. Therefore, it looks duplicated now.
On the other hand, I understand your point so we could move `setTag()` where you put it, but remove the previous call so it's not duplicated. Does it make sense?

Reply:
@nalcalag is not duplicated, those two setTag execute at very different moments in time.
the one in 276 executes when calling `addViewTreeObserverOnGlobalLayoutListener`. 
The on in 284 executes in the listener, when global layout changes.

In other words, line 276 sets the initial state, line 284 sets it for the updates